README.md

```diff
@@ -14,7 +14,6 @@ simply run:
 
 ```sh
 npm install
-grunt
```

Comment:
> how about defining both `prepublish` and `prepare`? If I read the deprecation note correctly, it would make the package able to work correctly across the npm versions, am I right?

I just tried defining both `prepublish` and `prepare`, and it causes the build to happen twice when `npm install` is run using npm v4.  So, it would work correctly across npm versions, but just takes a little extra time on npm v4, whereas npm v3 runs the build only once (as will npm v5 once it's released).

We can fix this by using [in-publish](https://www.npmjs.com/package/in-publish) to prevent the `prepublish` script from running upon `npm install` with npm v4. Alternatively, if all the developers who will be publishing this are already using npm v4, we can change `prepublish` to `prepublishOnly`, as detailed in the deprecation note linked in my previous comment.

> we should also take into account that the `devDependencies` are not retrieved when the package is installed as a dependency of another one, and so I'm not entirely sure that defining `prepublish`/`prepare` will be enough to get the the dist files correctly built in this scenario

Note that none of these scripts (`prepublish`, `prepare`, `prepublishOnly`) would be invoked when a user runs `npm install webextension-polyfill` (which is different from running `npm install` inside a clone of this repo). Since running `npm publish` invokes the `prepublish` script, the dist/ files will get built and included in the published package, so a user who runs `npm install webextension-polyfill` will have them without needing to install the `devDependencies`.

As I mentioned when I opened the PR, you can verify this by running the following:

```sh
npm pack && tar -tvf webextension-polyfill-0.1.0.tgz
```